src/pages/items/headsets/index.js

```diff
@@ -76,7 +76,7 @@ function Headsets() {
                 weight={1}
```

Comment:
Headsets sort buy weight is wrong, the ones that have two digits are considered to be the heaviest (like 0.43 is heavier than 0.7)